BUILD.bazel

```diff
@@ -171,41 +174,41 @@ gen_xchammer_config(
         },
         target_config = {
             "//:xchammer": xchammer_xcode_target_config,
-            "//tools/XCConfigExporter:xcconfig-exporter": xchammer_xcode_target_config
+            "//tools/XCConfigExporter:xcconfig-exporter": xchammer_xcode_target_config,
         },
-        targets = ["//:xchammer", "//tools/XCConfigExporter:xcconfig-exporter"],
+        targets = [
+            "//:xchammer",
+            "//tools/XCConfigExporter:xcconfig-exporter",
+        ],
     ),
 )
 
-# Xcode project for Bazel built project
-load(
-    "//:BazelExtensions/xcodeproject.bzl",
-    "xcode_project"
-)
-
 # The xcode_project for Bazel built Xcode project
 # Note: the target_config is declared in :XCHammerSourcesXcodeConfig
 xcode_project(
-    name="workspace_v2",
-    bazel="tools/bazelwrapper",
-    targets=["//:xchammer", "//tools/XCConfigExporter:xcconfig-exporter"],
-    project_config=project_config(
+    name = "workspace_v2",
+    bazel = "tools/bazelwrapper",
+    project_config = project_config(
+        generate_xcode_schemes = False,
         paths = ["**"],
-        generate_xcode_schemes=False,
         xcconfig_overrides = {
             "Release": "tools/BazelToolchain.xcconfig",
             "Debug": "tools/BazelToolchain.xcconfig",
             "Profile": "tools/BazelToolchain.xcconfig",
-        }
-    )
+        },
+    ),
+    targets = [
+        "//:xchammer",
+        "//tools/XCConfigExporter:xcconfig-exporter",
+    ],
 )
 
 # Development / Distribution
 genrule(
-    name = "xchammer_unzip",
-    srcs = [ ":xchammer" ],
+    name = "gen_xchammer_unzip",
```

Comment:
This is the main change from this file, the rest is formatting. This is just to get the rule and the output to have different names and resolve the bazel warning.